runtime/autoload/man.vim

```diff
@@ -135,7 +135,11 @@ function! s:get_page(path) abort
   let manwidth = empty($MANWIDTH) ? winwidth(0) : $MANWIDTH
   " Force MANPAGER=cat to ensure Vim is not recursively invoked (by man-db).
   " http://comments.gmane.org/gmane.editors.vim.devel/29085
-  return s:system(['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man', a:path])
+  if has('mac')
+    return s:system(['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man', a:path])
```

Comment:
actually, one more thing. can you change the above to:

```vim
  " Force MANPAGER=cat to ensure Vim is not recursively invoked (by man-db).
  " http://comments.gmane.org/gmane.editors.vim.devel/29085
  let cmd = ['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man']
  if has('mac')
    return s:system(cmd + [a:path])
  else
    return s:system(cmd + ['-l', a:path])
  endif
```

This will more clearly present the code that the comment is related to.